ztp/ztp-policy-generator/Makefile

```diff
@@ -0,0 +1,33 @@
+PWD := $(shell pwd)
+BASE_DIR := $(shell basename $(PWD))
+
+# Keep an existing GOPATH, make a private one if it is undefined
+GOPATH_DEFAULT := $(PWD)/.go
+export GOPATH ?= $(GOPATH_DEFAULT)
+GOBIN_DEFAULT := $(GOPATH)/bin
+export GOBIN ?= $(GOBIN_DEFAULT)
+export PATH := $(PATH):$(GOBIN)
+XDG_CONFIG_HOME=./
+KUSTOMIZE_PLUGIN_HOME ?= $(XDG_CONFIG_HOME)/kustomize/plugin
+
+.PHONY: build test test-kustomize all clean
+build:
+	go build -mod=vendor -o policygenerator/PolicyGenerator ./policygenerator/
+
+test:
+	go test ./policygenerator/...
```

Comment:
If we're doing `go test` here we should also remove it from the higher-level `ztp/Makefile`

(And as you can see from the `ci/prow/ci` results, that file needs to change anyway.)